src/main/resources/UMGCCity_DDL_master.sql

```diff
@@ -60,83 +65,87 @@ CREATE TABLE public.authorities
     authority character varying(50) COLLATE pg_catalog."default" NOT NULL
```

Comment:
Authorities table needs to change to mirror the authorities table in Prod. There are only two columns in this table now authority and id. The city_user table needs to be updated as well to include a authorities_id column that references the pk in authorities.